src/components/ec-inline-input-field/components/loading/loading.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div
+    class="tw-flex"
+    data-test="ec-inline-input-field-loading"
+  >
+    <div class="tw-flex tw-flex-grow tw-truncate tw-border tw-border-solid tw-border-gray-6 tw-rounded">
```

Comment:
this is chameleon, we should not use tw classes directly on elements, create BEM classes for it.